### ISSUE TITLE
[NLZiet] Error after latest update

### ISSUE THREAD

**Issue by FutureCow** [dut-iptv/dut-iptv.github.io#49]
With the newest version of the addon (form yesterday) i'm not able to use NLZiet anymore. I use Kodi 19 beta 2. Removed the addon and reinstalled, but doesnt fix the problem. Looking Live TV inside the addon is working. But when i try to watch with the Simple IPTV plugin I get this error message:

```
2020-12-21 09:48:20.041 T:4632    ERROR <general>: CCurlFile::Open failed with code 404 for https://nlzietprodstorage.blob.core.windows.net/static/channel-logos/npo1.pngrtl5.png:
                                                   
2020-12-21 09:48:20.083 T:4632    ERROR <general>: CCurlFile::Open failed with code 404 for https://nlzietprodstorage.blob.core.windows.net/static/channel-logos/npo1.pngsbs6.png:
                                                   
2020-12-21 09:48:20.123 T:4632    ERROR <general>: CCurlFile::Open failed with code 404 for https://nlzietprodstorage.blob.core.windows.net/static/channel-logos/npo1.pngrtl7.png:
                                                   
2020-12-21 09:48:20.169 T:4632    ERROR <general>: CCurlFile::Open failed with code 404 for https://nlzietprodstorage.blob.core.windows.net/static/channel-logos/npo1.pngveronica.png:
                                                   
2020-12-21 09:48:20.169 T:4144     INFO <general>: CPythonInvoker(1, C:\Users\fhett\AppData\Roaming\Kodi\addons\service.xbmc.versioncheck\resources\lib\runner.py): script successfully run
2020-12-21 09:48:20.201 T:4144     INFO <general>: Python interpreter stopped
2020-12-21 09:48:20.224 T:4632    ERROR <general>: CCurlFile::Open failed with code 404 for https://nlzietprodstorage.blob.core.windows.net/static/channel-logos/npo1.pngnet5.png:
                                                   
2020-12-21 09:48:21.259 T:6012     INFO <general>: Loading skin file: MyPVRChannels.xml, load type: KEEP_IN_MEMORY
2020-12-21 09:48:22.861 T:6012     INFO <general>: AddOnLog: pvr.iptvsimple: pvr.iptvsimple - PVRIptvData::GetChannelStreamProperties - Live Stream URL: plugin://plugin.video.nlziet/?_=play_video&channel=boW4lYH6CESbHzgZais_lQAAAAE&id=npo1&type=channel&pvr=1
2020-12-21 09:48:22.865 T:4864     INFO <general>: initializing python engine.
2020-12-21 09:48:23.137 T:4864    ERROR <general>: C:\Users\fhett\AppData\Roaming\Kodi\addons\script.module.fuzzywuzzy\lib\fuzzywuzzy\fuzz.py:11: UserWarning: Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning
                                                     warnings.warn('Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning')
                                                   
2020-12-21 09:48:23.592 T:4864  WARNING <general>: xbmc.translatePath is deprecated and might be removed in future kodi versions. Please use xbmcvfs.translatePath instead.
2020-12-21 09:48:23.592 T:4864  WARNING <general>: xbmc.translatePath is deprecated and might be removed in future kodi versions. Please use xbmcvfs.translatePath instead.
2020-12-21 09:48:24.159 T:4864    ERROR <general>: plugin.video.nlziet - local variable 'info' referenced before assignment
                                                   Traceback (most recent call last):
                                                     File "C:\Users\fhett\AppData\Roaming\Kodi\addons\plugin.video.nlziet\resources\lib\base\l5\signals.py", line 29, in throwable
                                                       yield
                                                     File "C:\Users\fhett\AppData\Roaming\Kodi\addons\plugin.video.nlziet\resources\lib\base\l6\router.py", line 90, in dispatch
                                                       function(**params)
                                                     File "C:\Users\fhett\AppData\Roaming\Kodi\addons\plugin.video.nlziet\resources\lib\base\l7\plugin.py", line 29, in decorated_function
                                                       item = f(*args, **kwargs)
                                                     File "C:\Users\fhett\AppData\Roaming\Kodi\addons\plugin.video.nlziet\resources\lib\base\l8\menu.py", line 666, in play_video
                                                       playdata = api_play_url(type=type, channel=channel, id=id, video_data=data, from_beginning=from_beginning, pvr=pvr)
                                                     File "C:\Users\fhett\AppData\Roaming\Kodi\addons\plugin.video.nlziet\resources\lib\api.py", line 372, in api_play_url
                                                       playdata = {'path': path, 'license': license, 'info': info, 'alt_path': alt_path, 'alt_license': alt_license}
                                                   UnboundLocalError: local variable 'info' referenced before assignment
2020-12-21 09:48:24.180 T:6012     INFO <general>: Loading skin file: DialogTextViewer.xml, load type: KEEP_IN_MEMORY
2020-12-21 09:48:39.112 T:4864     INFO <general>: CPythonInvoker(2, C:\Users\fhett\AppData\Roaming\Kodi\addons\plugin.video.nlziet\default.py): script successfully run
2020-12-21 09:48:42.259 T:6012     INFO <general>: Loading skin file: DialogButtonMenu.xml, load type: KEEP_IN_MEMORY
```

**Comment by oebeledrijfhout** [comment]
I'm having the same issue, trying to watch Live TV through the addon crashes Kodi for me. I'm using Libreelec 9.2.6, Kodi version 18.9.

**Comment by jbouwh** [comment]
Live tv still works using de plugin directly

**Comment by GuiSCI** [comment]
Comparable problems here. I have two similar systems ( Libreelec 9.2.6 - Kodi 18.9 ) and both no longer play NLZiet live streams. Not working directly in the add-on nor with PVR Simple IPTV. Replay TV and radio streams work fine.

![2020-12-24-215932](https://user-images.githubusercontent.com/67919391/103106221-dd006880-4633-11eb-9a82-303325d3540d.jpeg)

**Comment by dut-iptv** [comment]
Starting from 117 playback using Simple IPTV will be moved to a seperate addon which will also deal with this bug.

**Comment by FutureCow** [comment]
Thanks for the response, my wife if looking forward to it :)

**Comment by GuiSCI** [comment]
> Thanks for the response, my wife is looking forward to it :)

So is mine! LOL

**Comment by dut-iptv** [comment]
This bug was reproduced and fixed in 0.9.122 which is currently available in the beta repository.
The link between the addons and Simple IPTV have been moved to the "Dut-IPTV Simple IPTV Connector" addon.

If you want to keep using Simple IPTV to play content using my addons please update the addons you use to 0.9.122, re-login to your accounts and install the newly released addon "Dut-IPTV Simple IPTV Connector" and follow the setup wizard from the Program addons section of Kodi.

0.9.122 will probably be available in the main repository in a couple of days.

**Comment by FutureCow** [comment]
Thanks for this great update. I update the plugin + IPTV Connector. Looks like it's working nice. I do get a error after installing the IPTV connector addon, but it's working even with this error:


```
2021-01-10 22:24:00.212 T:5600    ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'KeyError'>
                                                   Error Contents: 'addon1'
                                                   Traceback (most recent call last):
                                                     File "C:\Users\user\AppData\Roaming\Kodi\addons\plugin.executable.dutiptv\service.py", line 2, in <module>
                                                       main()
                                                     File "C:\Users\user\AppData\Roaming\Kodi\addons\plugin.executable.dutiptv\resources\lib\service.py", line 15, in main
                                                       loop()
                                                     File "C:\Users\user\AppData\Roaming\Kodi\addons\plugin.executable.dutiptv\resources\lib\service.py", line 7, in loop
                                                       if api_get_all_epg() == True:
                                                     File "C:\Users\user\AppData\Roaming\Kodi\addons\plugin.executable.dutiptv\resources\lib\api.py", line 59, in api_get_all_epg
                                                       if len(profile_settings['addon' + unicode(x)]) > 0:
                                                   KeyError: 'addon1'
                                                   -->End of Python script error report<--
                                                   
2021-01-10 22:24:00.296 T:5600     INFO <general>: Python interpreter stopped
```
